pkg/virtual/framework/forwardingregistry/store.go

```diff
@@ -87,9 +92,115 @@ func DefaultDynamicDelegatedStoreFuncs(
 
 		return delegate.Get(ctx, name, *options, subResources...)
 	}
-	s.CreaterFunc = nil           // not currently supported
-	s.GracefulDeleterFunc = nil   // not currently supported
-	s.CollectionDeleterFunc = nil // not currently supported
+	s.CreaterFunc = func(ctx context.Context, obj runtime.Object, createValidation rest.ValidateObjectFunc, options *metav1.CreateOptions) (runtime.Object, error) {
+		err := rest.BeforeCreate(createStrategy, ctx, obj)
+		if err != nil {
+			return nil, err
+		}
+		if createValidation != nil {
+			if err := createValidation(ctx, obj.DeepCopyObject()); err != nil {
+				return nil, err
+			}
+		}
+		unstructuredObj, ok := obj.(*unstructured.Unstructured)
+		if !ok {
+			return nil, fmt.Errorf("not an Unstructured: %#v", obj)
+		}
+		delegate, err := client(ctx)
+		if err != nil {
+			return nil, err
+		}
+		return delegate.Create(ctx, unstructuredObj, *options, subResources...)
+	}
+	s.GracefulDeleterFunc = func(ctx context.Context, name string, deleteValidation rest.ValidateObjectFunc, options *metav1.DeleteOptions) (runtime.Object, bool, error) {
```

Comment:
what is deleteValidation in practice?

Reply:
`deleteValidation` is passed by `k8s.io/apiserver/pkg/endpoints/handlers/delete.go` and is `rest.AdmissionToValidateObjectDeleteFunc`, which is responsible to call admission logic.

Reply:
there is no admission in this context. Can we skip it and greatly simplify these funcs?

Reply:
You're right, there is no admission as it stands and I imagine there won't be any. I've removed the validation hooks in 4088a02ab0f75cdded8fe5e20e7b39f709e1438c.